docs/linking_to_external_code/reloading_modules.md

```diff
@@ -17,17 +17,17 @@ deno cache --reload my_module.ts
 Sometimes we want to upgrade only some modules. You can control it by passing an
 argument to a `--reload` flag.
 
-To reload all v0.55.0 standard modules
+To reload all **STD_VERSION** standard modules
```

Comment:
Why no underscores?

Reply:
oh, thats funny - `__` == `**` so prettier switched them out. ill change them all to `$STD_VERSION`